packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -22,7 +23,38 @@ export interface FileInfo {
   username: string;
   devHubUsername?: string;
 }
-export class OrgList {
+export class OrgList implements vscode.Disposable {
+  private statusBarItem: vscode.StatusBarItem;
+
+  constructor() {
+    this.statusBarItem = vscode.window.createStatusBarItem(
+      vscode.StatusBarAlignment.Left,
+      49
+    );
+    this.statusBarItem.command = 'sfdx.force.set.default.org';
+    this.statusBarItem.show();
+  }
+
+  public async showDefaultOrg(editor?: vscode.TextEditor) {
+    if (!editor) {
```

Comment:
Why do we need this check ? Currently, this will prevent the org picker from showing when you create a new project and customers would have to reload vscode in order to get this. 

This experience (not showing org picker) is not exclusive to creating a new project.

Reply:
Hmm, okay I went back and read the documentation and I'd understood it to mean that it's just looking for the text editor associated with the last touched source files. I assumed that this would be available in all contexts, unless something odd had happened; I didn't realize that a text editor isn't immediately set/there may not be source files available when you create a new project or update the extensions etc. We should be able to take this check out.